README.md

```diff
@@ -50,35 +50,92 @@ credentials as well as utility methods to create them and to get Application Def
 
 ## Quickstart
 
+### Alternative: using `java-libraries-bom`
```

Comment:
Perhaps not as `alternative` since it's the first option listed. I think we can probably put something like `Preferred/ Preference`?